File: 7 kyu/Bubble Sort Once/BubbleSortOnce.js
function bubblesortOnce(array) {
  let a = [...array];
  for (let i = 0; i<a.length-1; i++) {
    if (a[i] > a[i+1]) {
      let temp = a[i];
      a[i] = a[i+1];
      a[i+1] = temp;
    }
  }
  return a;
}

File: 7 kyu/Bubble Sort Once/BubbleSortOnce.cpp
std::vector<int> bubbleSortOnce(const std::vector<int> &input)
{
  std::vector<int> nums = input;
  for (int i = 0; i<nums.size()-1; i++) {
    if (nums[i] > nums[i+1]) {
      int temp = nums[i];
      nums[i] = nums[i+1];
      nums[i+1] = temp;
    }
  }
  return nums;
}
